qiskit_addon_sqd/fermion.py

```diff
@@ -240,7 +240,7 @@ def optimize_orbitals(
             k_flat, learning_rate, 0.9, num_steps_grad, dm1, dm2, hcore, eri_phys
         )
 
-    return e_qsci, k_flat, [np.diagonal(dm1), np.diagonal(dm1)]
+    return e_qsci, k_flat, [np.diagonal(dm1), np.diagonal(dm2_chem)]
```

Comment:
I believe you only reorder ``dm2`` to physicist's ordering to pass it to `optimize_orbitals_sci`. I believe we want to return the occs from `dm2_chem`, same as how we do in `solve_fermion`. Could you please verify, @jrm874 ?